prime-router/src/main/kotlin/cli/tests/SettingsTest.kt

```diff
@@ -88,24 +140,28 @@ class SettingsTest : CoolTest() {
             SettingCommand.SettingType.ORGANIZATION,
             settingName
         )
-
         /**
          * VERIFY the dummy organization existed or not
          */
         echo("VERIFY the dummy organization existed or not...")
 
-        val (_, _, result) = SettingsUtilities.get(path, dummyAccessToken)
+        val (_, _, result) = Fuel.get(testCase.path, testCase.parameters)
```

Comment:
Can you elaborate here?  I don't follow how this is checking if the organization exists or not?

Reply:
Ended up doing a bit of reworking to continue using the Utility functions instead